bitnami/nginx/templates/deployment.yaml

```
@@ -91,6 +91,7 @@ spec:
             - |
               [[ -f "/opt/bitnami/scripts/git/entrypoint.sh" ]] && source "/opt/bitnami/scripts/git/entrypoint.sh"
               git clone {{ .Values.cloneStaticSiteFromGit.repository }} --branch {{ .Values.cloneStaticSiteFromGit.branch }} /tmp/app
+              shopt -s dotglob
               [[ "$?" -eq 0 ]] && rm -rf /app/* && mv /tmp/app/* /app/
```

To be consistent with the `bitnami/apache` deployment template, could you please add it at the beginning and include the comment?

https://github.com/bitnami/charts/blob/73555a16d36e42d3301541abfc09b1810e2634c6/bitnami/apache/templates/deployment.yaml#L72-L80

```suggestion
            - |
              # Allow the mv to move hidden directories
              shopt -s dotglob
              [[ -f "/opt/bitnami/scripts/git/entrypoint.sh" ]] && source "/opt/bitnami/scripts/git/entrypoint.sh"
              git clone {{ .Values.cloneStaticSiteFromGit.repository }} --branch {{ .Values.cloneStaticSiteFromGit.branch }} /tmp/app
              [[ "$?" -eq 0 ]] && rm -rf /app/* && mv /tmp/app/* /app/
```